src/Integration.UnitTests/Telemetry/TelemetryDataRepositoryTests.cs

```diff
@@ -151,16 +151,24 @@ public void Can_Read_Old_TelemetryXml()
  <LastUploadDate>1999-12-31T23:59:59.9999999</LastUploadDate>
 </TelemetryData>");
 
+            // Calculate the expected result.
+            // Previously, this test started failing when daylight saving was applied on the test agent
+            // machine. Creating the local date first then converting it to a DateTimeOffset gives the
+            // expected result regardless of the local time zone or whether the test agent machine is
+            // automatically adjusting dor daylight saving time or not.
+            var expectedDate = new DateTime(1999, 12, 31, 23, 59, 59, 999, DateTimeKind.Local).AddTicks(9999);
+            var expectedDateTimeOffset = new DateTimeOffset(expectedDate);
```

Comment:
Actually, I've just looked at the other tests in this class and they are all creating the _DateTimeOffset_ by first creating a _DateTime_.